javascript/app.js

```diff
@@ -237,3 +242,23 @@ cocktailApp.revealButton.addEventListener("click", function () {
             // show recipe section
         });
 });
+
+cocktailApp.searchButton.addEventListener("click", function () {
```

Comment:
because this is a submit by nature we just need to add the prevent default

Reply:
added!